ionoscloud/resource_container_registry_token_test.go

```diff
@@ -0,0 +1,232 @@
+//go:build all || cr
+// +build all cr
+
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	cr "github.com/ionos-cloud/sdk-go-autoscaling"
+	"regexp"
+	"testing"
+)
+
+func TestAccContainerRegistryTokenBasic(t *testing.T) {
+	var containerRegistryToken cr.TokenResponse
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckContainerRegistryTokenDestroyCheck,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckContainerRegistryTokenConfigBasic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckContainerRegistryTokenExists(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, &containerRegistryToken),
+					// ToDo: fix diff between expiry_date post and get
+					//resource.TestCheckResourceAttr(ContainerRegistryTokenResource+"."+ContainerRegistryTokenTestResource, "expiry_date", "2023-01-13T16:27:42Z"),
```

Comment:
remove comment after deciding on a solution regarding the difference between the format sent at create and the one that is received at get